### ISSUE TITLE
NTR: Pachygyria with the cortical thickness > 10 mm

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4513]
Related to #4382
### 1. Preferred term label
NTR: Pachygyria with the cortical thickness > 10 mm

### 2. Synonyms
Classic lissencephaly, thick pachygyria; Type 1 lissencephaly

### 3. Textual definition 
Pachygyria with a very thick cerebral cortex measuring 10-20 mm. Note that cortical thickness cannot be measured reliably on scans done between 3 and 24 months of age.

COST:neuromig

Renske, how does this differ from the existing term, https://hpo.jax.org/app/browse/term/HP:0006818, Type I lissencephaly ?

**Comment by pnrobinson** [comment]
@Renske-Oegema  is this the same as Type I lissencephaly?
Where does 6-layer lissencephaly fit in?

**Comment by pnrobinson** [comment]
@Renske-Oegema  could you get back to me on this? Or could we skype about the remaining terms, there are a few things I am confused about.

**Comment by Renske-Oegema** [comment]
Pachygyria with the cortical thickness > 10 mm 
this term has 3 synonyms Classic lissencephaly, thick pachygyria; Type 1 lissencephaly
(the term type 1 lissencephaly is now obsolete, as type 2 has become cobblestone).

**Comment by pnrobinson** [comment]
adding as HP_0020187

**Comment by pnrobinson** [comment]
@Renske-Oegema 
Is this term the same thing as https://hpo.jax.org/app/browse/term/HP:0006818
Should we merge these terms?

**Comment by pnrobinson** [comment]
@Renske-Oegema I think these are synonyms or nearly so, could you please advise?

**Comment by pnrobinson** [comment]
I will close this for now, we can revisit at a later time point.
